libs/blocks/chart/chart.js

```diff
@@ -189,6 +190,7 @@ const handleIntersect = (chart, chartOptions) => (entries, observer) => {
 
   entries.forEach((entry) => {
     if (entry.isIntersecting) {
+      chart?.resize();
```

Comment:
Copying my comment from the previous PR: Ah, in that case, it seems like we shouldn't be initializing the chart ahead of time. Instead do the init in the observer handler

Reply:
Sounds good, updated this and consolidated chart initialization to its own function.